os_migrate/plugins/module_utils/server.py

```diff
@@ -14,13 +14,29 @@ class Server(resource.Resource):
 
     info_from_sdk = [
         'addresses',
+        'flavor',
```

Comment:
I tried this, but got an empty result for flavor_id, even though the right ID is filled out in flavor['id']. Would it be better to extend from_sdk to pick out just the ID from the flavor object instead of serializing the whole thing?

Reply:
Yes, and essentially the same thing can be done for turning `security_groups` into `security_group_ids`.

Reply:
Is this what you had in mind? https://github.com/os-migrate/os-migrate/pull/170/commits/9a6e5e46ac56032576664845bbae0ffb12ecdfbb

It does take flavor and security_groups out of the info section in the output YAML, but I'm not sure if I should be accessing obj.info directly like that.

Reply:
Yes, only one catch -- since we're adding `flavor_id` to `info` manually in the `from_sdk` override, we should remove it from the `info_from_sdk` parameter -- that one is being used to automatically copy properties of the sdk object, but we're doing it manually instead now. And if `flavor_id` isn't present in the sdk object directly as we've discussed above, that code would crash i think.

Reply:
Okay, I took it out.